tensorflow/python/framework/sparse_tensor.py

```diff
@@ -223,6 +223,63 @@ def shape(self):
     """
     return self._dense_shape_default
 
+  def set_shape(self, shape):
```

Comment:
This change has already been checked in? https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/framework/sparse_tensor.py#L226